sdcm/nemesis.py

```diff
@@ -1104,6 +1104,7 @@ def _get_kubernetes_node_break_methods(self):
         raise UnsupportedNemesis(f"Backend {self.cluster.params.get('cluster_backend')} "
                                  f"does not support node termination")
 
+    @decorate_with_context(ignore_ycsb_connection_refused)
```

Comment:
in case it will fail with connection refused, the load will stop, right?
is there another mechanism that will restart the load against another node?

Reply:
No, load isn't stopping cause of those errors.

The next request would refresh the DNS, and would go to the next node.